pyproject.toml

```diff
@@ -28,6 +28,8 @@ Nuitka = "^1.3.8"
 imageio = "^2.33.1"
 pytest = "^7.2.1"
 pytest-cov = "^4.1.0"
+pytest-mock = "^3.0"
```

Comment:
How is this mock different from the mock we previously used?

Reply:
needed to add this for the qt testing